src/libexpr/eval.hh

```diff
@@ -286,7 +286,7 @@ public:
     /* Print statistics. */
     void printStats();
 
-    void realiseContext(const PathSet & context);
+    void realiseContext(const PathSet & context, const Pos &pos, const string reason);
```

Comment:
This should be `const string & reason` (or `std::string_view reason` if you want to go C++20).